watson-developer-cloud/discovery-components
# chore: upgrade axios deps

#### What do these changes do/fix?
Implements https://github.ibm.com/Watson-Discovery/disco-issue-tracker/issues/8585

Updates `axios` to 0.21.4 and any dependencies that use `axios` to that version. One exception is the library `start-server-and-test`, which does not have a version released at the latest `axios` release. However, that library isn't used in production, so we should be fine as far as this vulnerability goes.


#### How do you test/verify these changes?
Make sure Cypress tests pass, and when you run `yarn why axios`, all resolved versions (except start-server-and-test` are at or above 0.21.4.
#### Have you documented your changes (if necessary)?
N/A
#### Are there any breaking changes included in this pull request?
Not as far as I checked in the dependencies.
